### ISSUE TITLE
[MultiSelect] Component cannot be used thru keyboard (Accessibility issue)

### ISSUE THREAD

**Issue by camilasb** [IBM/carbon-addons-iot-react#800]
Steps:
1. Open the Storybook
2. Open MultiSelect component (default sub-item)
3. Try to interact with the component using keyboard (TAB, ENTER, SPACE keys e.g.)
Actual Result: 2 TAB outs and user is taken to Show Indo icon at the right corner
Expected Result: User should be able to handle the component using the keyboard

**Comment by davidicus** [comment]
hi @camilasb. Thank you for opening this issue. Looking at our [storybook](https://carbon-addons-iot-react.com/?path=/story/multiselect--default) I am seeing keyboard controls for this component. I think because of the specific ux  it's not the common keyboard interaction of tabbing through each item. I do not know at this time if this is the standard approach to this type of input but I will look into it. 

For now, spacebar will open and close the multi-select, enter will choose an item  and tab will move to the next tab group. Can you confirm that this is the case? If you are still experiencing issues can you give us some more information on the type of OS and browser? Thanks in advance!

**Comment by camilasb** [comment]
Alright! With your instructions i'm able to interact with the component now.
I'll verify with the team if that's what's expected and will re-open it in case we need then.
Thanks! =)
